### PR TITLE
Disconnect on spinner unselect

### LINKED ISSUE #20: Muse/Sphero disconnection
When unselecting a Muse/Sphero from the select menu disconnect from it.

### PR DESCRIPTION
**Pull request by jmanart**
If from any of the spinners we unselect an option, unlink from the event bus. Closes #20 .

### CODE REVIEW

File: app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java [hunk 1]
```diff
@@ -265,6 +273,8 @@ public String apply(Robot bender) {
     }
 
     private void setEnabledStateForViews(@NonNull View[] views, boolean enabled) {
-        for (View view : views) { view.setEnabled(enabled); }
```
Comment (monchote) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java:268]: 👎 :(

File: app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java [hunk 2]
```diff
@@ -139,6 +139,7 @@ public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l)
             setEnabledStateForViews(mMuseActions, true);
 
             if (mSphero.isPresent()) {
+                if (mController.isPresent()) mController.get().unlink();
```
Comment (monchote) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: This line should probably go outside the outer if statement. Also, please add the curly brackets.

Comment (jmanart) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java]: I did intend to put it there.
I'll add the curly braces.

File: app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java (hunk 1 above)
Comment (jmanart) [inline app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java:268]: Auto format.

**Pull request merged**